readme.md

```diff
@@ -27,8 +27,22 @@ library.
 
 ## Example
 
+#### **`Cargo.toml`**
+
+```toml
+[dependencies]
+bincode = "1.2"
+serde =  { version = "1.0", features = ["derive"] }
```

Comment:
Since we're designing this for 2015 edition compilation, this doesn't need the derive feature and a dependency for `serde_derive` needs to be added

Reply:
Shouldn't two examples be made? One for consuming it with the 2015 edition and one for consuming it with the 2018 edition?

Reply:
`#[macro_use]` is still valid in the 2018 edition. And when we eventually get automated testing of the readme working we would have to make it 2015.